cmd/gator/test/test.go

```diff
@@ -72,20 +74,22 @@ func init() {
 	Cmd.Flags().StringVarP(&flagOutput, flagNameOutput, "o", "", fmt.Sprintf("Output format.  One of: %s|%s.", stringJSON, stringYAML))
 	Cmd.Flags().BoolVarP(&flagIncludeTrace, "trace", "t", false, "include a trace for the underlying Constraint Framework evaluation.")
 	Cmd.Flags().BoolVarP(&flagGatherStats, "stats", "", false, "include performance stats returned from the Constraint Framework.")
+	Cmd.Flags().BoolVarP(&flagEnableExternalData, "external-data", "e", false, "enable external_data function in Constraint Templates.")
+	Cmd.Flags().StringArrayVarP(&flagExternalDataProviders, "external-data-providers", "", []string{}, "a file containing External Data Provider. Can be specified multiple times.")
```

Comment:
I think the existence of `external-data-providers` implies that `external-data` is true so we could get away with just using the former flag.

Reply:
Similarly, probably don't need two options in `test.Opts`

Reply:
What is the reason for using a different flag to source providers vs. the rest of the cluster config? Avoiding letting untrusted config reach out to arbitrary HTTP endpoints?

Reply:
No specific reason. :) I agree with Alex's suggestion that the existence of `external-data-providers` implies that `external-data` is true.

Reply:
Gotcha. I think security is a good consideration as far as separating out the providers folder (at least potentially). And, if we do have a providers folder, that can probably take the place of an enabled/disabled flag. Maybe users who have a hard requirement of "absolutely no reaching out to external services" might take comfort in an explicit flag though.

@sozercan thoughts?

Reply:
@sozercan Kind reminder. :)

Reply:
+1 on having an explicit flag to enable/disable external data. though I prefer something like `enable-external-data`